src/components/requests/Requests.js

```diff
@@ -1,8 +1,15 @@
+/* eslint-disable array-callback-return */
```

Comment:
I can't comment on the specific line as this occured in a previous PR I think, but double check the `requests__subtitle` spacing on widths like 768px. It might make sense to center the text until the mobile width is reached?

Reply:
This is my 'fault' 😉 I will change it!